ansible/docker_site.yml

```diff
@@ -10,11 +10,18 @@
     - {role: ci/init/check_cache, when: run_build, tags: ["check_cache"]}
     - {role: ci/init/install, when: run_install, tags: ["install"]}
     - {role: ci/init/create_workspace, when: run_install, tags: ["create_workspace"]}
-    - {role: ci/update_dependencies, when: run_build, tags: ["update_dependencies", "build", "unit_tests", "code_coverage", "codecov_tool"]}
-    - {role: ci/build/catkin, when: run_build, tags: ["build", "unit_tests", "code_coverage", "codecov_tool"]}
-    - {role: ci/code_coverage/data, when: run_build, tags: ["code_coverage", "codecov_tool"]}
+    - {role: ci/update_dependencies, when: run_build, tags: ["update_dependencies", "build", "unit_tests",
+                                                             "code_coverage", "codecov_tool", "python_code_coverage",
+                                                             "cpp_code_coverage"]}
+    - {role: ci/build/catkin, when: run_build, tags: ["build", "unit_tests", "code_coverage", "codecov_tool",
+                                                      "python_code_coverage", "cpp_code_coverage"]}
+    - {role: ci/code_coverage/python/setup, when: run_build, tags: ["python_code_coverage", "code_coverage", "codecov_tool"]}
+    - {role: ci/code_coverage/cpp/setup, when: run_build, tags: ["cpp_code_coverage", "code_coverage", "codecov_tool"]}
     - {role: ci/unit_tests, when: run_build, tags: ["unit_tests", "code_coverage", "codecov_tool"]}
     - {role: ci/code_style_check/python, when: run_build, tags: ["python_code_style_check", "code_style_check"]}
     - {role: ci/code_style_check/cpp, when: run_build, tags: ["cpp_code_style_check", "code_style_check"]}
-    - {role: ci/servers/shippable/finalize, when: run_build, tags: ["shippable","shippable_finalize"]}
+    - {role: ci/servers/shippable/finalize, when: run_build, tags: ["shippable","shippable_finalize",
```

Comment:
remove from here  "python_code_coverage", "cpp_code_coverage". We do not want to run this role if server is not shippable